taskana-adapter/src/main/java/pro/taskana/adapter/impl/TaskanaTaskStarter.java

```diff
@@ -28,7 +28,8 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaTaskStarter.class);
 
-  @Autowired AdapterManager adapterManager;
+  @Autowired
+  AdapterManager adapterManager;
```

Comment:
something seems off with the formatter again

Reply:
done